### PR TITLE
Refactor scalardb-test

### PR DESCRIPTION
**Pull request by brfrn169**
What I did in this PR are as follows:

- Catch an exception and add a warning message for closing resources in `close()`
- Make `XXXWithRetry` classes inherit the same workload class without `WithRetry` to reduce code duplication.

Please take a look!

### CODE REVIEW

File: scalardb-test/src/main/java/kelpie/scalardb/transfer/graphql/HttpTransferProcessorWithRetry.java [hunk 1]
```diff
@@ -1,97 +1,25 @@
...
-import okhttp3.OkHttpClient;
-import okhttp3.Request;
-import okhttp3.RequestBody;
-import okhttp3.Response;
-import okhttp3.ResponseBody;
 
-public class HttpTransferProcessorWithRetry extends TimeBasedProcessor {
-
-  public static final MediaType JSON = MediaType.get("application/json");
+public class HttpTransferProcessorWithRetry extends HttpTransferProcessor {
```
Comment (brfrn169) [inline scalardb-test/src/main/java/kelpie/scalardb/transfer/graphql/HttpTransferProcessorWithRetry.java:11]: Made `HttpTransferProcessorWithRetry` inherit `HttpTransferProcessor`.

File: scalardb-test/src/main/java/kelpie/scalardb/transfer/TransferProcessorWithRetry.java [hunk 2]
```diff
@@ -1,38 +1,25 @@
...
 import io.github.resilience4j.core.IntervalFunction;
 import io.github.resilience4j.retry.Retry;
 import io.github.resilience4j.retry.RetryConfig;
-import java.util.Optional;
-import java.util.concurrent.ThreadLocalRandom;
 import java.util.concurrent.atomic.LongAdder;
 import javax.json.Json;
 
-public class TransferProcessorWithRetry extends TimeBasedProcessor {
+public class TransferProcessorWithRetry extends TransferProcessor {
```
Comment (brfrn169) [inline scalardb-test/src/main/java/kelpie/scalardb/transfer/TransferProcessorWithRetry.java:12]: Made `TransferProcessorWithRetry` inherit `TransferProcessor`.

File: scalardb-test/src/main/java/kelpie/scalardb/transfer/TransferWithTwoPhaseCommitTransactionProcessorWithRetry.java [hunk 3]
```diff
@@ -1,43 +1,28 @@
...
 import io.github.resilience4j.core.IntervalFunction;
 import io.github.resilience4j.retry.Retry;
 import io.github.resilience4j.retry.RetryConfig;
-import java.util.Optional;
-import java.util.concurrent.ThreadLocalRandom;
 import java.util.concurrent.atomic.LongAdder;
 import javax.json.Json;
 
-public class TransferWithTwoPhaseCommitTransactionProcessorWithRetry extends TimeBasedProcessor {
+public class TransferWithTwoPhaseCommitTransactionProcessorWithRetry
```
Comment (brfrn169) [inline scalardb-test/src/main/java/kelpie/scalardb/transfer/TransferWithTwoPhaseCommitTransactionProcessorWithRetry.java:14]: Made `TransferWithTwoPhaseCommitTransactionProcessorWithRetry` inherit `TransferWithTwoPhaseCommitTransactionProcessor`.

**Review by komamitsu — APPROVED** [review]
LGTM! 👍

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Review by brfrn169 — COMMENTED** [review]
It looks like I forgot to submit those inline comments...

**Review by Torch3333 — APPROVED** [review]
LGTM, thank you!

**Pull request merged**